server/src/main/java/io/spine/examples/shareaware/server/ShareAwareServer.java

```diff
@@ -0,0 +1,80 @@
+/*
+ * Copyright 2023, TeamDev. All rights reserved.
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ * http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Redistribution and use in source and/or binary forms, with or without
+ * modification, must retain the above copyright notice and the following
+ * disclaimer.
+ *
+ * THIS SOFTWARE IS PROVIDED BY THE COPYRIGHT HOLDERS AND CONTRIBUTORS
+ * "AS IS" AND ANY EXPRESS OR IMPLIED WARRANTIES, INCLUDING, BUT NOT
+ * LIMITED TO, THE IMPLIED WARRANTIES OF MERCHANTABILITY AND FITNESS FOR
+ * A PARTICULAR PURPOSE ARE DISCLAIMED. IN NO EVENT SHALL THE COPYRIGHT
+ * OWNER OR CONTRIBUTORS BE LIABLE FOR ANY DIRECT, INDIRECT, INCIDENTAL,
+ * SPECIAL, EXEMPLARY, OR CONSEQUENTIAL DAMAGES (INCLUDING, BUT NOT
+ * LIMITED TO, PROCUREMENT OF SUBSTITUTE GOODS OR SERVICES; LOSS OF USE,
+ * DATA, OR PROFITS; OR BUSINESS INTERRUPTION) HOWEVER CAUSED AND ON ANY
+ * THEORY OF LIABILITY, WHETHER IN CONTRACT, STRICT LIABILITY, OR TORT
+ * (INCLUDING NEGLIGENCE OR OTHERWISE) ARISING IN ANY WAY OUT OF THE USE
+ * OF THIS SOFTWARE, EVEN IF ADVISED OF THE POSSIBILITY OF SUCH DAMAGE.
+ */
+
+package io.spine.examples.shareaware.server;
+
+import io.spine.base.Production;
+import io.spine.server.Server;
+import io.spine.server.ServerEnvironment;
+import io.spine.server.delivery.Delivery;
+import io.spine.server.storage.memory.InMemoryStorageFactory;
+import io.spine.server.transport.memory.InMemoryTransportFactory;
+
+import java.io.IOException;
+
+import static io.spine.client.ConnectionConstants.DEFAULT_CLIENT_SERVICE_PORT;
+
+/**
+ * A {@code gRPC} server running a "Trading" bounded context.
+ */
+public class ShareAwareServer {
+
+    /**
+     * Prevents instantiation of this class.
+     */
+    private ShareAwareServer() {
+    }
+
+    /**
+     * The entry point of the ShareAware application.
+     */
+    public static void main(String[] args) throws IOException {
+        var server = create();
+        server.start();
+        server.awaitTermination();
+    }
+
+    /**
+     * Creates {@code Server} instance at the
+     * {@linkplain io.spine.client.ConnectionConstants#DEFAULT_CLIENT_SERVICE_PORT default} port.
+     */
+    static Server create() {
+        configureEnvironment();
+        var context = TradingContext.newBuilder();
+        return Server
+                .atPort(DEFAULT_CLIENT_SERVICE_PORT)
+                .add(context)
+                .build();
+    }
+
+    private static void configureEnvironment() {
+        ServerEnvironment
+                .when(Production.class)
+                .use(InMemoryStorageFactory.newInstance())
```

Comment:
We need to let users know about this somehow. I suggest to mention that in class-level documentation.

Otherwise it's not too clear why we lose all of our data upon each restart.